### PR TITLE
fix(learn): clarify gradient description

### PR DESCRIPTION
**Pull request by jennifernorris**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

The description of the 45 degree CSS linear-gradient states that it is angled like a backslash, which is confusing because it's actually angled like a forward slash. I'm suggesting a modification of the description that I think is clearer.

### CODE REVIEW

**Comment by Sky020** [comment]
Hello there,

I would say the current description is correct. Here is an example:
image

So, going `left-right`, and `red -> blue`, you get a `\` (backslash) with `45deg`.

I will leave this for others to weigh-in, if you do not agree.

**Comment by jgarst37** [comment]
I see what you are saying, as the apparent line produced by the gradient pattern does look like a backslash. However, the 90deg example is described as being horizontal, but following the same logic, it is actually a vertical line:

image

The question is whether the description should be of the direction of the gradient, or the direction of the apparent line it creates. I think either choice is acceptable, but it should be consistent between the two examples to avoid confusion. I brought it up because it _did_ confuse me.

**Comment by nhcarrigan** [comment]
I'll chime in!
image

Here is a 0 degree gradient - which, according to MDN, is described as going from bottom to top. So, I think our 90 degree description is *correct* and our 45 degree description should be updated.

**Review by naomi-lgbt — DISMISSED** [review]
LGTM~! 🥳 I agree that the descriptions are not consistent, and I think this is the more accurate change.

Congratulations on your first pull request to this project! We look forward to future contributions!

As a note, our workflow does require at least one additional approval before this can be merged.

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md:13]: I agree with adding the `(from bottom left to top right)`, but if you look at @Sky020's example, that looks like a backslash and not a forward slash, so I think the `backslash` part is more accurate.

Comment (nhcarrigan) [reply curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md:13]: IMHO, I'd remove the slash reference entirely.

A 90 degree gradient creates a vertical line, with colours shifting across the horizontal axis. So this is accurate:
> where <code>90deg</code> makes a horizontal gradient (from left to right)

A 45 degree gradient creates a diagonal line `\` with colours shifting from bottom left to top right. I think saying it is angled "like a backslash" or "angled like a forward slash" both create a bit of confusion, and could be changed to mirror the 90 degree description:
> and <code>45deg</code> makes a diagonal gradient (from bottom left to top right)

Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md:13]: >and `45deg` makes a diagonal gradient (from bottom left to top right)

Sounds good to me.

Comment (nhcarrigan) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md:13]:
```suggestion
The first argument specifies the direction from which color transition starts - it can be stated as a degree, where <code>90deg</code> makes a horizontal gradient (from left to right) and <code>45deg</code> makes a diagonal (from bottom left to top right). The following arguments specify the order of colors used in the gradient.
```

@jgarst37 could you apply this change?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

@RandellDawson or @Sky020 would you mind taking another look?

**Review by RandellDawson — APPROVED** [review]
LGTM.  @moT01 What do you think?

**Review by moT01 — APPROVED** [review]
LGTM @jgarst37 🎉 Congratulations on your first contribution to this freeCodeCamp repository 🎉 We welcome future contributions 😄

**Pull request merged**
